terasoluna-tourreservation-selenium/src/test/java/org/terasoluna/tourreservation/tourreserve/selenium/TokenCheckErrorTest.java

```diff
@@ -261,7 +264,11 @@ public void testReserveUpdateToken() {
                 "_TRANSACTION_TOKEN")).clear();
 
         // change reservation
-        driver.findElement(By.id("changeBtn")).click();
+        jse.executeScript("document.getElementById('changeBtn').click();");
+
+        Wait<WebDriver> wait = new WebDriverWait(driver, 4, 2000);
```

Comment:
- [ ] Why you wait individually? Is `WebDriverEventListener#afterScript` not work?

Reply:
As you said, this process was not necessary if ` WebDriverEventListener#afterScript` was run.
So, implemented ` WebDriverEventListener#afterScript` and  run wait processing.

Since the wait condition is the same as that implemented by afterClickOn,
we implement a common wait processing method `waitProcessing`.
173a4d0